moto/stepfunctions/models.py

```diff
@@ -176,6 +208,104 @@ def __init__(
         self.status = "RUNNING"
         self.stop_date = None
 
+    def get_execution_history(self, roleArn):
+        execution_history_type = os.environ.get("EXECUTION_HISTORY_TYPE", "SUCCESS")
```

Comment:
This should probably move to settings.py, probably prefixed with `SFN_` to make it more obvious where it's used.

As to documentation: We don't really have any similar environment variables, or a good place to document them. Maybe just add a comment to settings.py with a short explanation.

Reply:
@bblommers does `settings.py` exist already? Or are you asking me to make one for Step Functions specifically?

Reply:
It already exist: `moto/settings.py`

Reply:
🙃  Wow VS Code search failed me there. Cool will do!

Reply:
Okay so doing that makes actually setting it using mock.patch a lot harder - is there a recommended way to change a setting in a unit test style?

I guess this problem will exist when trying to use moto also.

Reply:
As the value is assigned as soon as 'settings' is imported, the consumer of `moto` is stuck with whatever value it assigns first

Reply:
Ah, good point. The other settings don't need to be recomputed during testing, so this is a first.

Easiest way to get around that is probably to just make it a function, instead of a property:
`def get_..(): return os.environ.get(.., ..)`

The alternatives would probably be to either:
 - patch settings.py:property, if that's possible
 - or reload the settings.py (using importlib.reload)

Neither are pretty IMO

Reply:
If we're happy with it as a function in `settings.py` I can certainly do that! I'll put it in and push it up!